src/GitVersionExe/GitVersion.Tool.csproj

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <PackAsTool>true</PackAsTool>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+    <ToolCommandName>dotnet-gitversion</ToolCommandName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
nitpick: Guessing this `GenerateAssemblyInfo` prop can be deleted here, as it's already imported from GitVersionExe.csproj ?

Reply:
You're right, I'll change that